test/crud_tests.py

```diff
@@ -23,8 +23,10 @@
 from struct import *
 from __builtin__ import *
 
+TEST_DB_NAME = 'sample database'
```

Comment:
Please make TEST_DB_NAME local to class similar to how host and masterKey are defined.